cpu/msp430-common/include/cpu_conf.h

```diff
@@ -25,7 +25,7 @@ extern "C" {
 #endif
 
 #define THREAD_STACKSIZE_IDLE      (96)
-#define MSP430_ISR_STACK_SIZE           (256)
+#define ISR_STACKSIZE      (256)
```

Comment:
Please align value with the value in the line above?

Also: Why has native it's own commit for that, but not the rest?